.github/workflows/ci.yaml

```diff
@@ -329,6 +329,11 @@ jobs:
     runs-on: ${{ github.repository_owner == 'coder' && 'depot-ubuntu-22.04-8' || 'ubuntu-latest' }}
     needs:
       - changes
+    strategy:
+      matrix:
+        POSTGRES_VERSION:
+          - "13"
+          - "16"
```

Comment:
We should ensure only one of these is required in the `required` job to avoid adding two chances for flakes to prevent merge. I don't expect there to be many (or any) situations where one passes and the other fails because of an actual problem.

Reply:
Won't this allow merging despite consistent failures on either variant of the matrix?

Reply:
Yeah, but do we expect this to happen ever? If the matrix only requires 13, we have reasonable confidence it should work in 16. Requiring both in CI gives more chance for flakes to block merge.

Reply:
OK, I'm going to make this a separate job instead then. Using a matrix strategy makes the `required` check more complicated than I'd like.